class-auth.php

```diff
@@ -351,14 +355,15 @@ public function validate_token( $return_response = true ) {
 		list($token) = sscanf( $auth, 'Bearer %s' );
 
 		if (!$token) {
-            // Check if Basic Auth is used and create a token if so
-            list($username, $password) = explode( ':', base64_decode( substr( $auth, 6 ) ) );
-            $request = new WP_REST_Request( 'POST', '/wp-json/jwt-auth/v1/token' );
-            $request->set_param( 'username', $username );
-            $request->set_param( 'password', $password );
-            $token = $this->generate_token( $request );
-            if (is_array($token) && isset($token['token'])) $token = $token['token'];
-            return;
+            return new WP_REST_RESPONSE(
+                array(
+                    'success'       => true,
+                    'statusCode'    => 200,
+                    'code'          => 'basic_auth_valid',
+                    'message'       => __('Basic auth is valid', 'jwt-auth'),
+                    'data'          => array()
+                )
+            );
         }
 
 		if ( ! $token ) {
```

Comment:
Hard to follow in the PR column but didn't we just send a response for `!$token` ? Maybe the JWT stuff can be encapsulated in it's own function if we are nesting too far here.

Reply:
Thanks for catching this, didn't realize there was a duplicate if